Rjegl08/switch sample links (#2054)
capire/docs: get-started/in-a-nutshell.md

```
:::

> *Find this source also in `cap/samples` [for Node.js](https://github.com/capire/bookshop/blob/main/db/schema.cds), and [for Java](https://github.com/SAP-samples/cloud-cap-samples-java/blob/main/db/books.cds)*{ .learn-more}

Author: ```suggestion
*Find this source also in `capire/bookshop` [for Node.js](https://github.com/capire/bookshop/blob/main/db/schema.cds), and [for Java](https://github.com/SAP-samples/cloud-cap-samples-java/blob/main/db/books.cds)*{ .learn-more}
```

[Find the full set of options here.](../tools/cds-cli.md#data){.learn-more}
:::

> [Find a full set of `.csv` files in **cap/samples**.](https://github.com/capire/bookshop/tree/main/db/data){ .learn-more target="_blank"}

Author: ```suggestion
[Find a full set of `.csv` files in **capire/bookshop**.](https://github.com/capire/bookshop/tree/main/db/data){ .learn-more target="_blank"}
```

### Vue.js UIs {#vue .impl .node}

Besides Fiori UIs, CAP services can be consumed from any UI frontends using standard AJAX requests.

> For example, you can [find a simple Vue.js app in **cap/samples**](https://github.com//capire/bookshop/tree/main/app/vue), which demonstrates browsing and ordering books using OData requests to [the `CatalogService` API we defined above](#services). {.impl .node}

Author: ```suggestion
For example, you can [find a simple Vue.js app in **capire/bookshop**](https://github.com//capire/bookshop/tree/main/app/vue), which demonstrates browsing and ordering books using OData requests to [the `CatalogService` API we defined above](#services). {.impl .node}
```

└─ ...
```

> [See these files also in **cap/samples**/bookshop/srv folder.](https://github.com//capire/bookshop/tree/main/srv){.learn-more}

Author: ```suggestion
[See these files also in **capire/bookshop** in the _srv_ folder.](https://github.com//capire/bookshop/tree/main/srv){.learn-more}
```

</span>

> [Find this source also in **cap/samples**.](https://github.com//capire/bookshop/tree/main/srv/cat-service.js){ .learn-more .impl .node target="_blank"}

Author: ```suggestion
[Find this source also in **capire/bookshop**.](https://github.com//capire/bookshop/tree/main/srv/cat-service.js){ .learn-more .impl .node target="_blank"}
```
